pandas/tests/frame/test_axis_select_reindex.py

```diff
@@ -117,6 +117,26 @@ def test_drop(self):
         df.drop(labels=df[df.b > 0].index, inplace=True)
         assert_frame_equal(df, expected)
 
+    def test_drop_empty_index(self):
+        # GH 27994
+        df_index = pd.DataFrame(
+            {"a": [0, 1]},
+            index=[pd.Timestamp("2019-01-01"), pd.Timestamp("2019-01-01")],
+        )
+        assert_frame_equal(df_index.drop([]), df_index)
```

Comment:
use the pattern

```
result=
expected=
assert_frame_equatl(result, expected)
```

if expected doesn't change you don't need to repeat it

Reply:
Changed in a new commit.